plugins/bright_plugin/build/Dockerfile

```diff
@@ -12,9 +12,9 @@ COPY utils/ ${BASE_PATH}/utils/
 COPY ${SRC_BASE_DIR}/conf/supervisord.conf /etc/supervisor/conf.d/
 COPY ${SRC_BASE_DIR}/scripts/init.sh ${SRC_BASE_DIR}/scripts/deinit.sh /
 
-RUN apt-get update && apt-get -y install supervisor python3.9 python3-pip vim curl sudo
+RUN apt-get update && apt-get -y install supervisor python3 python3-pip vim curl sudo
 
-RUN python3.9 -m pip install -r ${BASE_PATH}/${SRC_BASE_DIR}/src/bright/requirements.txt
+RUN python3 -m pip install -r ${BASE_PATH}/${SRC_BASE_DIR}/src/bright/requirements.txt
```

Comment:
why did you remove the Python version 3.9?

Reply:
The installed version of python3.9 has [high vulnerability severity](http://hpc-master.lab.mtl.com:8080/view/Security/job/ANCHORE-SCAN-DOCKER-New/357/artifact/logs/mswg_release_ufm_plugins_bright_ufm-plugin-bright_1.0.0-3-docker.img.gz/bright-1.0.0-3.html)

upgrading to the base ubuntu22 and installing python3 only will install by default the python3.10 which works fine with the bright plugin packages